docs/usage/usage-guide.md

```diff
@@ -230,30 +231,6 @@ updateValue(state, action) {
 
 Much better!
 
-### Defining Functions in Objects
```

Comment:
This paragraph doesn't make sense here any more. I've moved it down to `createSlice`, but I'm not 100% happy with the position there. Maybe we could also get it removed, because the Computed property notation is not really relavant any more and the rest needs fewer explanation?